tests/by-util/test_stdbuf.rs

```diff
@@ -1,13 +1,71 @@
+#[cfg(target_os = "linux")]
 use crate::common::util::*;
 
+#[cfg(target_os = "linux")]
```

Comment:
Do you know if:
```suggestion
#[cfg(not(windows))]
```
would work?

Reply:
unless I'm missing something
```rust
#[cfg(not(target_os = "windows"))]
```
seems to work.